processor/marketplace_processor/offer/offer_acceptance.py

```diff
@@ -93,87 +94,161 @@ def check_validity_of_offer(offer, accept_offer):
 
 class OfferAcceptance(object):
 
-    def __init__(self, offer, accept_offer, state):
+    def __init__(self, offer, accept_offer, header, state):
         self._offer = offer
         self._accept_offer = accept_offer
+        self._header = header
```

Comment:
This is unused

Reply:
We could take it out here. It will be used in the EXCHANGE_ONCE and EXCHANGE_ONCE_PER_ACCOUNT PR. It is a pain though since it is my commit in @dplumb94's PR.